airbyte-webapp/src/components/ColumnSortButton/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+import styles from "./ColumnSortButton.module.scss";
+import SortIcon from "./SortIcon";
+
+interface ColumnSortButtonProps {
+  onClick: () => void;
+  formattedMessageId: string;
+  wasActive: boolean;
+  lowToLarge: boolean;
+}
+
+export const ColumnSortButton: React.FC<ColumnSortButtonProps> = ({
+  onClick,
+  formattedMessageId,
+  wasActive,
+  lowToLarge,
+}) => (
+  <button className={styles.sortButton} onClick={onClick}>
+    <FormattedMessage id={formattedMessageId} />
+    <SortIcon wasActive={wasActive} lowToLarge={lowToLarge} />
```

Comment:
Is it necessary to have a sort icon component file? It looks like it's very simple and it could be implemented here.